docs/source/2_getting_started/2_getting_started.rst

```diff
@@ -1,2 +1,30 @@
 Getting Started
 ===============
+
+A good way to get started is looking at one of the example scripts.
+
+Examples
+--------
+
+Example scripts are located under
+`examples <https://github.com/shimming-toolbox/shimming-toolbox-py/tree/master/examples>`__.
+
+general_demo
+^^^^^^^^^^^^
+
+This example shows the process of using the toolbox with dicom data and processing them to output an unwrapped phase
+plot. More precisely, it will:
+
+* Download unsorted dicoms
+* Run dcm2bids to convert to nifti with bids structure
+* Unwrap phase
+* Save wrapped and unwrapped plot of first X,Y volume as unwrap_phase_plot.png in the output directory
+
+To use the script, simply provide an output directory where the processing and output figure will be generated, if no
+directory is provided, a directory *output_dir* will be created in the current directory.
+
```

Comment:
I'm having trouble to run sphinx locally, could you describe the procedure above in a wiki page on GH and i'll fill it in with my findings.

Reply:
Issue was that I installed sphinx with pip3 instead of pip so the command `python -msphinx` did not work. Could we go for `python3 -msphinx` instead since the project requires python > 3.6

Reply:
Oops, I neglected to add this installation step in the docs: in order to build the package locally with sphinx as a dependency, you can install the package (inside your activated miniconda virtualenv) with `shimming-toolbox-py: $ pip install -e .[testing,docs]`. I'll add this as an issue.

This *should* add `sphinx` to your path, allowing the `docs/Makefile` to function correctly.

Reply:
Alternatively, once you push the commit to this branch remote, you can see the resulting build on RTD here: https://shimming-toolbox-py--46.org.readthedocs.build/en/46/. The link is available in the same list of checks where the Travis branch build link is located:

![Screen Shot 2020-08-06 at 16 07 22](https://user-images.githubusercontent.com/12765385/89577616-f62c8800-d7fe-11ea-9b1a-4d65b75307f1.png)

Reply:
Thanks for the info. I really like that we can build locally so we don't over commit. I'll take a look at your PR and it should fix the issues.